codeserver/ubi9-python-3.9/Pipfile

```diff
@@ -0,0 +1,47 @@
+[[source]]
+name = "pypi"
+url = "https://pypi.org/simple"
+verify_ssl = true
+
+[[source]]
+name = "pytorch"
+url = "https://download.pytorch.org/whl/cu118"
+verify_ssl = true
+
+[dev-packages]
+
+[packages]
+# Base packages
+wheel = "~=0.41.2"
+setuptools = "~=68.1.2"
+
+# Datascience packages
+boto3 = "~=1.29.7"
+kafka-python = "~=2.0.2"
+matplotlib = "~=3.8.2"
+numpy = "~=1.26.2"
+pandas = "~=2.1.3"
+plotly = "~=5.18.0"
+scikit-learn = "~=1.3.2"
+scipy = "~=1.11.4"
+skl2onnx = "~=1.15.0"
+codeflare-sdk = "~=0.12.1"
+kfp = "~=1.8.22"
+kfp-tekton = "~=1.8.1"
+ipykernel = "~=6.26.0"
+
+# PyTorch packages
+tensorboard = "~=2.15.1"
+torch = {version = "~=2.1.1", index = "pytorch"}
```

Comment:
any reason for only installing torch and not tensorflow.
on hindsight , i feel we should not install any of these big packages.
and only stick with basic packages like

boto3, matplotlib, pandas, numpy, scipy

Reply:
yes, there is a similar issue in @guimou recent changes to the contrib datascience UI pathon packages, they have all sorts of heavy nvidia* libraries in it, regardless of whether the base docker image is for Tensorflow or not. https://github.com/opendatahub-io-contrib/workbench-images/issues/48. Agreed that in a first step here, minimal might be good. And you could get these changes here in this PR into the snippets for codeserver VSCode [at contrib](https://github.com/opendatahub-io-contrib/workbench-images/tree/main), so users can then always add more of their own packages, if they want to, in own builds.

Reply:
answerered in contrib https://github.com/opendatahub-io-contrib/workbench-images/issues/48#issuecomment-1842844803

Reply:
Thank you for reviewing this! 😊 I was in a dilemma about whether to include the most advanced packages or not. I agree with you; I'll omit them for now and stick to the basic data science  packages.